kolibri/auth/models.py

```diff
@@ -69,6 +68,34 @@ class FacilityDataSyncableModel(SyncableModel):
     class Meta:
         abstract = True
 
+    def serialize(self):
```

Comment:
I just copied the logic from the generic serialization and deserialization in morango. Added custom logic for handling the DateTimeTZField.
Maybe we could just handle the custom logic and call the parent class method.